content/en/_index.html

```diff
@@ -28,10 +28,13 @@
   <div class="container">
     <h3 class="section-head">What is Kubeflow?</h3>
     <p class="mx-auto col-md-8 px-0">
-      The Kubeflow project is dedicated to making deployments of machine learning (ML) workflows on Kubernetes simple, portable and scalable.
-      Our goal is not to recreate other services, but to provide a straightforward way to deploy best-of-breed open-source systems for ML to diverse infrastructures.
+      Kubeflow makes artificial intelligence and machine learning simple, portable, and scalable.
+      We are an <i>ecosystem</i> of <a href="https://kubernetes.io/" target="_blank">Kubernetes</a>
+      based components for each stage in 
+      <a href="/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle" target="_blank">the AI/ML Lifecycle</a> 
+      with support for best-in-class open source <a href="/docs/started/architecture/#kubeflow-ecosystem" target="_blank">tools and frameworks</a>.
       <br><br>
-      Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+      Deploy <a href="/docs/started/installing-kubeflow/" target="_blank">Kubeflow</a> anywhere you run Kubernetes.
```

Comment:
I think we should move the "Deploy" within the link so it's clear what the link is.

Reply:
Sure